bin/tools/Templates/FrontendUnitTest.js

```diff
@@ -665,6 +903,123 @@ export class FrontendUnitTest extends AbstractTemplate {
    * @returns {string}
    * @constructor
    */
+  static get MODELS_TEST_TPL() {
+    let content = [];
+
+    content.push(`// THIS TEST WAS GENERATED AUTOMATICALLY ON ${new Date().toString()}`);
+    content.push('');
+    content.push('\'use strict\';');
+    content.push('');
+    content.push('{import}');
+    content.push('');
+    content.push('// @todo: Add more advanced tests');
+    content.push('describe(\'Models\', () => {');
+    content.push('');
+    content.push('  describe(\'{ClassName}\', () => {');
+    content.push('    it(\'Class {ClassName} exists\', () => {');
+    content.push('      expect(typeof {ClassName}).toBe(\'function\');');
+    content.push('    });');
+    content.push('');
+    content.push('    it(\'Check {ClassName} constructor\', () => {');
+    content.push('      let {objectName} = new {ClassName}({});');
+    content.push('    expect({objectName} instanceof {ClassName}).toBeTruthy();');
+    content.push('    });');
+    content.push('  });');
+    content.push('');
+    content.push('});');
+    content.push('');
+
+    return content.join(os.EOL);
+  }
+
+  /**
+   * @returns {string}
+   * @constructor
+   */
+  static get MODELS_TEST_TPL_WITH_SERVICE() {
```

Comment:
Use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals for such strings, instead of array.join(os.EOL)